firmware/controllers/idle_thread.cpp

```diff
@@ -184,7 +193,23 @@ static float autoIdle() {
 		targetRpm = interpolate2d("cltRpm", clt, CONFIG(cltIdleRpmBins), CONFIG(cltIdleRpm), CLT_CURVE_SIZE);
 	}
 
-	percent_t newValue = idlePid.getValue(targetRpm, getRpmE(engine), engineConfiguration->idleRpmPid.period);
+	// check if within the dead zone
+	int rpm = getRpmE(engine);
+	if (absI(rpm - targetRpm) <= CONFIG(idlePidRpmDeadZone))
+		return baseIdlePosition;
+
+	// When rpm < targetRpm, there's a risk of dropping RPM too low - and the engine dies out.
+	// So PID reaction should be increased by adding extra percent to PID-error:
+	percent_t errorAmpCoef = 1.0f;
+	if (rpm < targetRpm)
+		errorAmpCoef += (float)CONFIG(pidExtraForLowRpm) / PERCENT_MULT;
+	// If errorAmpCoef > 1.0, then PID thinks that RPM is lower than it is, and controls IAC more aggressively
+	idlePid.setErrorAmplification(errorAmpCoef);
+
```

Comment:
And this is an error amplification coef. implementation.
See https://github.com/rusefi/rusefi/pull/589/commits/016084a76c8d2c4c9bb7bef0f21f637c316c5524